app/resources/create.html.template

```diff
@@ -377,37 +378,12 @@
               <div class="end-multi-columns"></div>
             </div>
           {% endif %}
-
-          <div class="expiry section">
-            <h2><label for="expiry_option">
-                {% trans "Expiry" %}
-            </label></h2>
-            <div class="hint">
-              <div>
-                {% trans "When should this record disappear?" %}
-              </div>
-              {% if env.repo_test_mode %}
-                <div class="warning">
-                  {% trans "While test mode is in effect, records that are over 6 hours old are deleted regardless of the expiry date." %}
-                </div>
-              {% endif %}
-            </div>
-            <div class="field">
-              <select id="expiry_option" name="expiry_option">
-                {% for duration in env.expiry_options %}
-                <option value="{{duration.value}}">
-                  {{duration.text}}
-                </option>
-                {% endfor %}
-              </select>
-            </div>
-            <div class="end-multi-columns"></div>
           </div>
 
-          <div class="source section">
-            <h2>
+          <div class="sourcerecord section">
```

Comment:
Would be fine to keep it "source" here. Otherwise make it "source-of-record". "sourcerecord" sounds rather like a record of a source.